sdk/core/core-paging/src/models.ts

```diff
@@ -62,3 +62,8 @@ export interface PagedResult<TPage, TPageSettings = PageSettings, TLink = string
    */
   toElements?: (page: TPage) => unknown[];
 }
+
+export type Paged<T> = {
+  value: T[];
+  nextLink?: string;
```

Comment:
Could we add docstrings for the properties too?

Reply:
added